x-pack/test/security_solution_api_integration/test_suites/detections_response/utils/rules/prebuilt_rules/install_prebuilt_rules_fleet_package.ts

```diff
@@ -23,49 +31,71 @@ export const installPrebuiltRulesFleetPackage = async ({
   supertest,
   version,
   overrideExistingPackage,
+  retryService,
 }: {
   es: Client;
   supertest: SuperTest.SuperTest<SuperTest.Test>;
   version?: string;
   overrideExistingPackage: boolean;
-}): Promise<void> => {
+  retryService: RetryService;
+}): Promise<InstallPackageResponse | BulkInstallPackagesResponse> => {
   if (version) {
     // Install a specific version
-    await supertest
-      .post(epmRouteService.getInstallPath('security_detection_engine', version))
-      .set('kbn-xsrf', 'true')
-      .send({
-        force: overrideExistingPackage,
-      })
-      .expect(200);
+    const response = await retry<InstallPackageResponse>({
+      test: async () => {
+        const testResponse = await supertest
+          .post(epmRouteService.getInstallPath('security_detection_engine', version))
+          .set('kbn-xsrf', 'true')
+          .send({
+            force: overrideExistingPackage,
+          })
+          .expect(200);
+        expect((testResponse.body as InstallPackageResponse).items).toBeDefined();
+        expect((testResponse.body as InstallPackageResponse).items.length).toBeGreaterThan(0);
+
+        return testResponse.body;
+      },
+      retryService,
+      retries: 2,
+      timeout: 120000,
```

Comment:
Same comments for this file as for `utils/rules/prebuilt_rules/install_fleet_package_by_url.ts`.